### PR TITLE
Fix true disp calculation

### PR DESCRIPTION
**Pull request by MaxNoe**
The true disp and sign calculation used a wrong definition
of the azimuthal angle in polar coordinates (cog - source) instead
of (source - cog).

This was only working, because this introduces a sign, which was
corrected by calculating the sign accordingly.

No corrected and simplified.

### CODE REVIEW

File: aict_tools/preprocessing.py [hunk 1]
```diff
@@ -37,25 +37,20 @@ def check_valid_rows(df):
...
-        true_disp = euclidean_distance(
-            source_x, source_y,
-            cog_x, cog_y
-        )
-
-        true_delta = np.arctan2(
-            cog_y - source_y,
-            cog_x - source_x,
-        )
-        true_sign = np.sign(np.abs(delta - true_delta) - np.pi / 2)
```
Comment (LukasNickel) [inline aict_tools/preprocessing.py:50]: So this saved us before?

File: aict_tools/preprocessing.py [hunk 2]
```diff
@@ -37,25 +37,20 @@ def check_valid_rows(df):
...
-
-        return true_disp, true_sign
-
     delta_x = source_x - cog_x
     delta_y = source_y - cog_y
 
-    # in the projected case, true disp is the coordiante of the source
-    # on the long axis
+    # in the projected case,
+    # true disp is the coordiante of the source on the long axis
```
Comment (LukasNickel) [inline aict_tools/preprocessing.py:44]: *coordinate

File: aict_tools/preprocessing.py [hunk 3]
```diff
@@ -37,25 +37,20 @@ def check_valid_rows(df):
...
     true_disp = np.cos(delta) * delta_x + np.sin(delta) * delta_y
+    sign_disp = np.sign(true_disp)
+
+    if project_disp is False:
+        abs_disp = euclidean_distance(
```
Comment (LukasNickel) [inline aict_tools/preprocessing.py:49]: I think its somewhat confusing that true_disp is the same for both cases, but the absolute disp differs. I can see why the code is like this, just wanted to point it out.

File: aict_tools/preprocessing.py (hunk 1 above)
Comment (MaxNoe) [reply aict_tools/preprocessing.py:50]: Yes, basically it was 180degrees difference to the usual definition of the sign

File: aict_tools/preprocessing.py (hunk 3 above)
Comment (MaxNoe) [reply aict_tools/preprocessing.py:71]: Added explanation in the docstring

**Pull request merged**